main.tf

```diff
@@ -0,0 +1,81 @@
+# To store the deployment state on S3 and use dynamodb for locking:
+# - deploy with ./deploy
+# - uncomment section below and update the bucket name
```

Comment:
Why is this step needed?

Reply:
You need to bootstrap the bucket before you can actually use it.

Reply:
No monads in terraform, of course :)

It might be better to move these steps into README, feels weird instructions are at two places (it was confusing to me)

Reply:
Certainly no MonadFix :D
Will do.